lib/rules/await-async-events.ts

```diff
@@ -3,6 +3,7 @@ import { ASTUtils, TSESLint, TSESTree } from '@typescript-eslint/utils';
 import { createTestingLibraryRule } from '../create-testing-library-rule';
 import {
 	findClosestCallExpressionNode,
+	findClosetFunctionExpression,
```

Comment:
Typo in the function name: s/Closet/Closest

Reply:
Also a bit of an inconsistency here, seems like this should be `findClosestFunctionExpressionNode` to match `findClosestCallExpressionNode`?

Reply:
I think we have covered these cases, right? @epmatsw

Reply:
Oops, wrong comment. But yes, also addressed